plugins/MauticSocialBundle/Views/Monitoring/leads.html.php

```diff
@@ -0,0 +1,102 @@
+<?php
+/**
+ * @package     Mautic
+ * @copyright   2016 Mautic, Inc. All rights reserved
+ * @author      Mautic, Inc
+ * @link        https://mautic.org
+ * @license     GNU/GPLv3 http://www.gnu.org/licenses/gpl-3.0.html
+ */
+?>
+
+<div class="pa-md bg-auto">
+    <?php if (count($items)): ?>
+        <div class="shuffle-grid row">
+            <?php foreach ($items as $item): ?>
+                <div class="shuffle shuffle-item card grid col-sm-6 col-lg-4">
+                    <div class="panel ovf-h" style="border-top: 3px solid <?php // echo $item['color']; ?>;">
+                        <div class="box-layout">
+                            <div class="col-xs-4 va-m">
+                                <div class="panel-body">
+                                <span class="img-wrapper img-rounded" style="width:100%">
+                                    <?php $preferred = $item['preferred_profile_image']; ?>
+                                    <?php if ($preferred == 'gravatar' || empty($preferred)) : ?>
+                                        <?php $img = $view['gravatar']->getImage($item['email'], '250'); ?>
+                                    <?php else : ?>
+                                        <?php $socialData = unserialize($item['social_cache']); ?>
+                                        <?php $img = (!empty($socialData[$preferred]['profile']['profileImage'])) ? $socialData[$preferred]['profile']['profileImage'] : $view['gravatar']->getImage($item['email'], '250'); ?>
+                                    <?php endif; ?>
+                                    <img class="img img-responsive" src="<?php echo $img; ?>" />
+                                </span>
+                                </div>
+                            </div>
+                            <div class="col-xs-8 va-t">
+                                <div class="panel-body">
+                                    <?php if (in_array($item['id'], $noContactList)) : ?>
+                                        <div class="pull-right label label-danger"><i class="fa fa-ban"> </i></div>
+                                    <?php endif; ?>
+                                    <h4 class="fw-sb mb-xs">
+                                        <a href="<?php echo $view['router']->generate('mautic_lead_action', array("objectAction" => "view", "objectId" => $item['id'])); ?>" data-toggle="ajax">
```

Comment:
This is probably where it's failing for me:
```
app/cache/prod/appProdUrlGenerator.php line 205 {"exception":"[object] (Symfony\\Component\\Routing\\Exception\\RouteNotFoundException(code: 0): Unable to generate a URL for the named route \"mautic_lead_action\" as such route does not exist. at /var/www/m.johnlinhart.com/public_html/app/cache/prod/appProdUrlGenerator.php:205)"} []
```
![mautic-social](https://cloud.githubusercontent.com/assets/1235442/16147977/eeb2a4d0-3486-11e6-92a5-13252c336cde.png)